src/v2/imports/carts/bigcommerce.ts

```diff
@@ -6,11 +6,19 @@ const bigcommerceTracker = ({ appId, retailId }: Pick<QueryStringContext, "appId
         try {
             const transaction = JSON.parse(JSON.stringify(JSON.parse(xhr.responseText)));
             const products = transaction?.lineItems?.physicalItems;
+<<<<<<< HEAD
```

Comment:
can we address the conflicts, thanks

Reply:
That's the thing I don't know how to address it. I'm not sure which code is correct here. Wondering if you could resolve it

Reply:
Conflict resolve